app/views/lobby/home.scala

```diff
@@ -47,11 +47,20 @@ object home:
             "lobby-nope" -> (playban.isDefined || currentGame.isDefined || homepage.hasUnreadLichessMessage)
           )
         )(
-          div(cls := "lobby__table")(
+          div(cls := "lobby__buttons")(
+            div(cls := "lobby__counters")(a, a),
             div(cls := "lobby__start")(
               button(cls := "button button-metal", tpe := "button", trans.site.createAGame()),
               button(cls := "button button-metal", tpe := "button", trans.site.playWithAFriend()),
               button(cls := "button button-metal", tpe := "button", trans.site.playWithTheMachine())
```

Comment:
these are still using the previous texts. Old translations should be deleted to ensure they're not used anymore